### ISSUE TITLE
Inconsistent behavior of ScaleShiftRotate on masks

### ISSUE THREAD

**Issue by psFournier** [albumentations-team/albumentations#877]
## 🐛 Bug

With np.int64 mask inputs, ScaleShiftRotate produces seemingly at random int64 and int32 outputs.
## To Reproduce

```
import numpy as np
from torch.utils.data import Dataset, DataLoader
import albumentations as A

aug = A.ShiftScaleRotate()
class RandomDataset(Dataset):

  def __getitem__(self, index):
      image = np.random.randint(0,255, size=(128,128,3), dtype=np.uint8)
      label = np.random.randint(0, 255, size=(128,128,3), dtype=np.int64)
      res = aug(image=image, mask=label)
      return res['image'], res['mask']

  def __len__(self):
          return 2

ds = DataLoader(RandomDataset(), batch_size=2)
for epoch in range(20):
    print("epoch {}".format(epoch))
    for batch in ds:
        image, label = batch
        print(label.dtype)
```
produces a random mix (different each time) of int64 and int32 mask batches:
```
epoch 0
torch.int32
epoch 1
torch.int64
epoch 2
torch.int64
epoch 3
torch.int64
epoch 4
torch.int32
epoch 5
torch.int64
epoch 6
torch.int64
epoch 7
torch.int64
epoch 8
torch.int64
epoch 9
torch.int64
epoch 10
torch.int64
epoch 11
torch.int32
...
```
After further inspection, it is highly possible that this behavior also holds for ElasticTransform, which points to a conversion happening during spatial transformations (perhaps during interpolation ?).

## Expected behavior

Always return batches of int64 masks.

## Environment

 - Albumentations 0.5.2
 - Python 3.7
 - Linux
 - How you installed albumentations (`conda`, `pip`, source): pip

**Comment by BloodAxe** [comment]
Could be related to `cv2.warpAffine` support of int64 dtype: https://stackoverflow.com/a/55696736/387739

**Comment by Dipet** [comment]
Do not use `int64` for masks and images. Library works incorrectly and very slow with this type of images.
I think that you don't need all `2^64` combinations of values for mask.
If you need `int64` dtype for mask just convert it is after transformation.

Looks like we need to add check that image and mask does not have `int64` or `uint64` dtype.

**Comment by psFournier** [comment]
Hello,
Categorical targets in torch must be LongTensor and thus require int64 inputs it seems, so yeah i convert my transformed image right before passing them to the loss. 
A warning in these transfo would be great, simply to avoid a weird half-reproducible bug... (i haven't checked inside opencv why this behavior appears).
